### DISCUSSION TITLE
Make the targets in Traget Practice mod "shatter" upon click (in legacy skins)

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#23350] [Ideas]
This cutting-edge exclusive animation is missing from Lazer, probably due to the default lazer-hardcoded skins having their own unique hit animations.

cutting-edge:

https://user-images.githubusercontent.com/75596775/235363921-534367de-3ac7-4279-8d5f-cab2057efac2.mp4

lazer:

https://user-images.githubusercontent.com/75596775/235363930-98fd230e-9135-4f6b-bd30-17a3e739b1e5.mp4

https://user-images.githubusercontent.com/75596775/235364103-eb9ddd58-6aa7-4c67-9fee-67085d01e1e1.mp4

**Comment by bdach** [comment]
Not sure how feasible this is to make happen.

**Reply by Joehuu** [reply]
Related: https://github.com/ppy/osu/discussions/22660#discussioncomment-5186814

**Comment by peppy** [comment]
Personally I'd hope we can bring back the targets (override user skin, default on but as a mod setting).
